crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -8,14 +8,44 @@ import "./interfaces/ICommonBridge.sol";
 /// @title CommonBridge contract.
 /// @author LambdaClass
 contract CommonBridge is ICommonBridge, Ownable, ReentrancyGuard {
-    constructor(address owner) Ownable(owner) {}
+    /// @notice Mapping of unclaimed withdrawals. A withdrawal is unclaimed if
+    /// there is a non-zero value in the mapping (a merkle root) for the hash
+    /// of the L2 transaction that requested the withdrawal.
+    /// @dev The key is the hash of the L2 transaction that requested the
+    /// withdrawal.
+    /// @dev The value is a boolean indicating if the withdrawal was claimed or not.
+    mapping(bytes32 => bool) public claimedWithdrawals;
```

Comment:
Maybe this should be `unclaimedWithdrawals`
```suggestion
    mapping(bytes32 => bool) public unclaimedWithdrawals;
```

Reply:
Then I think we should change the docs:
```
/// @notice Mapping of unclaimed withdrawals. A withdrawal is unclaimed if
/// there is a non-zero value in the mapping (a merkle root) for the hash
/// of the L2 transaction that requested the withdrawal.
```
It should be "A withdrawal is claimed if there is a non-zero value..."

Reply:
Done [191eb47](https://github.com/lambdaclass/lambda_ethereum_rust/pull/944/commits/191eb47b78ab4dba1ee4ff478750d5428afee2ad)